linode/linode_client.py

```diff
@@ -226,6 +226,25 @@ def get_users(self, *filters):
         """
         return self.client._get_and_filter(User, *filters)
 
+def create_user(self, email, username, password, restricted=True):
+    """
+    Creates a user
+    """
+    params = {
+        "email": email,
+        "username": username,
+        "password": password,
+        "restricted": restricted
+    }
+    result = self.client.post('/account/users', data=params)
+
+    if not 'email' and 'restricted' and 'username' in result:
```

Comment:
I usually only check for the primary key to see if the response was what I expected - populate is pretty tolerant of missing values, so by only requiring the primary key we prevent api changes from breaking clients who need an upgrade. That being said, I would be surprised if the user object lost any of those attributes.